### ISSUE TITLE
Show redaction reason (safely)

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#18581]
Messages can be redacted with a reason. We should show it like the mobile apps do, being sure to not let long/insane reasons cause problems.

**Comment by SimonBrandner** [comment]
Related: https://github.com/vector-im/element-web/issues/17246

**Comment by bkil** [comment]
* Redaction reasons should only be specified and shown when redacting the message of someone else (i.e., by moderators).
* If moderators are causing abuse within a given room, that room is forsaken by definition, so you shouldn't worry about that.

**Comment by SimonBrandner** [comment]
> Redaction reasons should only be specified and shown when redacting the message of someone else (i.e., by moderators).

I disagree, sometimes I want to send a redaction even if I am not a moderator, explaining why I deleted the message, so that people don't get confused

> If moderators are causing abuse within a given room, that room is forsaken by definition, so you shouldn't worry about that.

You'd still probably want to avoid showing the harmful content to the user

**Comment by Half-Shot** [comment]
Closing this. It's confusing to have an open issue about showing redaction reasons and another calling for it's removal. Given #17246 has been tagged against product, I'm going to close this and we can open again if the resolution is to include them.
